Makefile

```diff
@@ -106,3 +106,8 @@ install: $(GOOS)
 .PHONY: uninstall
 uninstall: $(GOOS)
 	-rm $(DESTDIR)/$(PROJECT)
+
+.PHONY: e2e-test
+e2e-test: 
+	- make linux
+	- ./bin/linux/e2e verify -c ./test/e2e/e2e.yaml 
```

Comment:
1. We should not assume that developers are using linux
2. Should be `run`, instead of `verify`